output.tf

```diff
@@ -1,3 +1,22 @@
+output "address" {
+  value = aws_db_instance.mz_rds_demo_db.address
```

Comment:
Hmm, maybe this should be `host` in line with the Postgres naming convention (e.g. `$PGHOST`).

Reply:
I think it's fine to use the RDS instance terminology here, since this is a pretty direct wrapper of the RDS resource. Think you want to output `port` too though, as AFAICT `address` contains only the hostname?

Reply:
An option here would be to output the whole `aws_db_instance.mz_rds_demo_db` object. That way the user would have all of the properties and they could decide which one they want to use in their extended modules.

Reply:
> Think you want to output port too though, as AFAICT address contains only the hostname?

Yes, I forgot about that because I'm using the default in my module.

> An option here would be to output the whole aws_db_instance.mz_rds_demo_db object. 

@bobbyiliev can you push a suggestion in this PR so I can try it out in [my benchmarking repo](https://github.com/MaterializeInc/letrec-bench) before we merge this?

Reply:
Yep! Just made the suggestion bellow. Let me know what you think.